code/datums/mood_events/generic_positive_events.dm

```diff
@@ -203,3 +203,8 @@
 	description = "<span class='nicegreen'>Truly, that was the food of the Divine!</span>\n"
 	mood_change = 5
 	timeout = 3 MINUTES
+
+/datum/mood_event/bipolar_happy
+	description = "<span class='nicegreen'>I'm feeling a bit more happy than usual.</span>\n"
+	mood_change = 6
```

Comment:
is bipolar_happy intended to have half of the magnitude of bipolar_sad?

Reply:
Tbh most of it is just the same as jolly/depression (unless I managed to copy it wrong)
I could lower the effect of bipolar_sad, but checking the code, depression mild has a bigger effect
Although lower number might make it easier to pass it off as a neutral quirk